data/anaconda.conf

```diff
@@ -60,6 +60,15 @@ system_root = /mnt/sysroot
 # Should we install the network configuration?
 can_configure_network = True
 
+# Should we copy input kickstart to target system?
+copy_input_kickstart = True
```

Comment:
I would follow the name pattern for system and target properties:

```
can_copy_input_kickstart
can_save_output_kickstart
```

There might be more logs to copy in the future, so I would change the last one into:
```
can_copy_installation_logs
```

Reply:
Sounds good to me, will do.